js/import/import.ui.js

```diff
@@ -855,7 +860,7 @@ const attachListeners = () => {
                       // auto generate transformation config
                       const mapping = getImporterSectionsMapping(originalURL) || [];
                       transform = TransformFactory.create(
-                        buildTransformationConfigFromMapping(mapping),
+                          buildTransformationRulesFromMapping(mapping)
```

Comment:
We should common lint'ing rules I think.  Mine will probably not like that indenting and will request a trailing comma.

Reply:
I had eslint disabled in my IDE. It is now enabled which will hopefully help.